tasks/hyrax-metadata-updates/tests/test-integration-index.js

```diff
@@ -312,3 +313,108 @@ test('Test generate path from ECHO-10 throws exception with broken config', asyn
     message: 'Provider not supplied in configuration. Unable to construct path'
   });
 });
+
+test.serial('Test updating ECHO10 metadata file in S3 when formats are present and correct', async (t) => {
+  // Set up mock Validation call to CMR
+  nock('https://cmr.earthdata.nasa.gov').post('/ingest/providers/GES_DISC/validate/granule/MOD11A1.A2017200.h19v04.006.2017201090724.cmr.xml')
+    .reply(200);
+  await setupS3(t, false, 'tests/data/echo10inWithFormat.xml');
+  const config = cloneDeep(event.config);
+  config.hyrax = {
+    formats: [
+      'HDF-5'
+    ]
+  };
+  const e = {
+    config: config,
```

Comment:
Completely unimportant, but in later versions of Node, if you are assigning a variable to a key of the same name in an object, you can just do:

```
{
  config,
  ...
}
```